usaspending_api/transactions/delta_models/published_fabs.py

```diff
@@ -0,0 +1,120 @@
+    "modified_at": {"delta": "TIMESTAMP", "postgres": "TIMESTAMP"},
+    "non_federal_funding_amount": {"delta": "NUMERIC(23,2)", "postgres": "NUMERIC(23,2)"},
+    "original_loan_subsidy_cost": {"delta": "NUMERIC(23,2)", "postgres": "NUMERIC(23,2)"},
+    "period_of_performance_curr": {"delta": "STRING", "postgres": "TEXT"},
+    "period_of_performance_star": {"delta": "STRING", "postgres": "TEXT"},
+    "place_of_perfor_state_code": {"delta": "STRING", "postgres": "TEXT"},
+    "place_of_perform_country_c": {"delta": "STRING", "postgres": "TEXT"},
+    "place_of_perform_country_n": {"delta": "STRING", "postgres": "TEXT"},
+    "place_of_perform_county_co": {"delta": "STRING", "postgres": "TEXT"},
+    "place_of_perform_county_na": {"delta": "STRING", "postgres": "TEXT"},
+    "place_of_perform_state_nam": {"delta": "STRING", "postgres": "TEXT"},
+    "place_of_perform_zip_last4": {"delta": "STRING", "postgres": "TEXT"},
+    "place_of_performance_city": {"delta": "STRING", "postgres": "TEXT"},
+    "place_of_performance_code": {"delta": "STRING", "postgres": "TEXT"},
+    "place_of_performance_congr": {"delta": "STRING", "postgres": "TEXT"},
+    "place_of_performance_forei": {"delta": "STRING", "postgres": "TEXT"},
+    "place_of_performance_zip4a": {"delta": "STRING", "postgres": "TEXT"},
+    "place_of_performance_zip5": {"delta": "STRING", "postgres": "TEXT"},
+    "place_of_performance_scope": {"delta": "STRING", "postgres": "TEXT"},
+    "record_type": {"delta": "INTEGER", "postgres": "INTEGER"},
+    "record_type_description": {"delta": "STRING", "postgres": "TEXT"},
+    "sai_number": {"delta": "STRING", "postgres": "TEXT"},
+    "submission_id": {"delta": "NUMERIC(23,2)", "postgres": "NUMERIC(23,2)"},
+    "total_funding_amount": {"delta": "STRING", "postgres": "TEXT"},
+    "uei": {"delta": "STRING", "postgres": "TEXT"},
+    "ultimate_parent_legal_enti": {"delta": "STRING", "postgres": "TEXT"},
+    "ultimate_parent_uei": {"delta": "STRING", "postgres": "TEXT"},
+    "ultimate_parent_unique_ide": {"delta": "STRING", "postgres": "TEXT"},
+    "unique_award_key": {"delta": "STRING", "postgres": "TEXT"},
+    "updated_at": {"delta": "TIMESTAMP", "postgres": "TIMESTAMP"},
+    "uri": {"delta": "STRING", "postgres": "TEXT"},
+}
+PUBLISHED_FABS_DELTA_COLUMNS = {k: v["delta"] for k, v in PUBLISHED_FABS_COLUMNS.items()}
+PUBLISHED_FABS_POSTGRES_COLUMNS = {k: v["postgres"] for k, v in PUBLISHED_FABS_COLUMNS.items()}
+
+published_fabs_create_sql_string = fr"""
+    CREATE OR REPLACE TABLE {{DESTINATION_TABLE}} (
+        {", ".join([f'{key} {val}' for key, val in PUBLISHED_FABS_DELTA_COLUMNS.items()])}
+    )
+    USING DELTA
+    LOCATION 's3a://{{SPARK_S3_BUCKET}}/{{DELTA_LAKE_S3_PATH}}/{{DESTINATION_DATABASE}}/{{DESTINATION_TABLE}}'
+"""
+
+published_fabs_load_sql_string = fr"""
+    INSERT OVERWRITE {{DESTINATION_DATABASE}}.{{DESTINATION_TABLE}}
+    (
+        {",".join([col for col in PUBLISHED_FABS_DELTA_COLUMNS])}
+    )
+"""
```

Comment:
Same as the other PR, no need to include it if not used.

Reply:
removed